helm-chart/banzai/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.1.1"
+appVersion: "1.1.2"
 description: A Helm chart to deploy the BANZAI pipeline
 name: banzai
-version: 1.1.0
+version: 1.1.1
```

Comment:
I dislike that these versions are not in lock step.

Reply:
Made the helm chart version 1.1.2 to match the application version. Will keep them in lockstep from now on.